components/user_settings/modal/user_settings_modal.jsx

```diff
@@ -116,7 +118,7 @@ class UserSettingsModal extends React.Component {
         }
 
         if (this.state.active_tab !== prevState.active_tab) {
-            $(ReactDOM.findDOMNode(this.refs.modalBody)).scrollTop(0);
+            $(ReactDOM.findDOMNode(this.modalBodyRef.current)).scrollTop(0);
```

Comment:
You don't need the `indDOMNode` at all, you only need the `$(this.modalBodyRef.current).scrollTop(0)`

Reply:
Another thing that we try to do is to remove any jQuery usage when is possible, in this case is easy, i think, probably something like `this.modalBodyRef.current.scrollTop = 0` would work (I'm not 100% sure, but I think will work in all the browsers, including IE11).

Reply:
Sorry, my fault, you are completely right, when you get a reference to a component, you receive the component, not the underneath element.